doc/Example-Gradle-Project/gradle/wrapper/gradle-wrapper.jar.license

```diff
@@ -0,0 +1,7 @@
+This file is part of CPAchecker,
```

Comment:
First, do we really need a JAR file checked in here?

Second, this seems to a file from Gradle, but its `.license` file claims something completely different. Very likely copyright and license info are wrong?